index.js

```diff
@@ -0,0 +1,27 @@
+const { Command } = require('commander');
+const program = new Command();
```

Comment:
`command` will be a better name. 😅 Easier to understand.

Reply:
Actually this is the [standard way](https://github.com/tj/commander.js/#declaring-program-variable) recommended by the commanderJS documentation. Should I change it?